### ISSUE TITLE
DEBUG node doesn't exist yet when the docs mention it's use. 

### ISSUE THREAD

**Issue by caspercasanova** [JarkkoPar/Utility_AI_GDExtension#19]
The [Docs](https://github.com/JarkkoPar/Utility_AI_GDExtension/blob/main/tutorial/Getting_started_with_the_Node_Query_System.md#52-adding-a-point-grid-and-spawning-to-the-main-scene) mention adding a `DEBUG` node yet the I don't see its existence in my downloaded asset library.

Using Godot version v4.2.2

Also perhaps creating a discord might help or a discussions section. 

EDIT: I might be an idiot. Noticed that the node is a node 2d not a debug node. -_-

**Comment by JarkkoPar** [comment]
Yes, you only need to set the name of the node as ”DEBUG”, and you can use a Node2D or any derived node (and Node3D for 3D). For 2D a Sprite2D node would work fine. The purpose of the debug node is to make the points visible so that you can see how your point grid is doing.
